src/ralph/admin/sitetrees.py

```diff
@@ -10,6 +10,7 @@
 )
 
 from ralph.admin.sites import ralph_site
+from ralph.data_importer.management.commands.importer import APP_MODELS
```

Comment:
it's only one line so we can redefine it here. besides that - it's definition (`APP_MODELS = {model._meta.model_name: model for model in apps.get_models()}`) breaks when there will be model with the same name in more than one app